### PR TITLE
fix "Public key not valid (Point not on curve)" error

### PR DESCRIPTION
**Pull request by bvavala**
tlcc now checks whether the local attestation (from C code) has failed and outputs an error message.
Without the check, in the case of attestation error, tlcc still passes the parameters ecc, which parses the enclave's public key and fails because the key is likely not valid.

### CODE REVIEW

File: tlcc/enclave/enclave_stub.go [hunk 1]
```diff
@@ -19,6 +19,8 @@ package enclave
 import (
 	"unsafe"
 
+    "fmt"
+
```
Comment (mbrandenburger) [inline tlcc/enclave/enclave_stub.go]: no empty line

Comment (mbrandenburger) [reply tlcc/enclave/enclave_stub.go]: can you run goimport please

File: tlcc/enclave/enclave_stub.go [hunk 2]
```diff
@@ -101,11 +103,14 @@ func (e *StubImpl) GetLocalAttestationReport(targetInfo []byte) ([]byte, []byte,
 	defer C.free(targetInfoPtr)
 
 	// call enclave
-	// TODO read error
-	C.tlcc_get_local_attestation_report(e.eid,
+    ret := C.tlcc_get_local_attestation_report(e.eid,
```
Comment (mbrandenburger) [inline tlcc/enclave/enclave_stub.go]: formatting; run gofmt to fix that

File: tlcc/enclave/enclave_stub.go [hunk 3]
```diff
@@ -101,11 +103,14 @@ func (e *StubImpl) GetLocalAttestationReport(targetInfo []byte) ([]byte, []byte,
...
 		(*C.target_info_t)(targetInfoPtr),
 		(*C.report_t)(reportPtr),
 		(*C.ec256_public_t)(pubkeyPtr))
+    if ret != 0 { // 0 is SGX_SUCCESS
```
Comment (mbrandenburger) [inline tlcc/enclave/enclave_stub.go]: formatting; run gofmt to fix that

**Comment by bvavala** [comment]
replaced by #40

**Pull request closed without merging**